lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py

```diff
@@ -0,0 +1,119 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+import unittest
+
+import yaml
+from integ_tests.s1aptests import s1ap_wrapper
+from s1ap_utils import InitMode, MagmadUtil
+from yaml.loader import SafeLoader
+
+SERVICE_START_TIME = 'start_time'
+SERVICE_ACTIVE = 'active_status'
+SERVICE_RESULT = 'result'
+
+
+class TestServicesAreRunning(unittest.TestCase):
+    """
+    A simple smoke test that checks if all relevant magma services are running.
+    This test was introduced because of GH14007 where a service that is not used
+    in the integration tests was damaged in the magma debian package artifact.
+    """
+
+    def setUp(self):
+        """Initialize"""
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper(
+            health_service=MagmadUtil.health_service_cmds.ENABLE,
+        )
+
+    def test_services_are_running(self):
+
+        if self._s1ap_wrapper.magmad_util.init_system.value != InitMode.SYSTEMD.value:
+            self.skipTest("Systemd only test.")
+
+        services = self._get_services()
+
+        # initialize service status dictionary
+        service_status = {
+            service: {
+                SERVICE_START_TIME: [],
+                SERVICE_ACTIVE: [],
+                SERVICE_RESULT: [],
+            } for service in services
+        }
+
+        self._query_state_of_services(services, service_status)
```

Comment:
`services` can be removed as an argument (also in other calls below) - the information is already in the `service_status` dict. will do the change depending on the discussion if the new test actually provides a benefit.

Reply:
done